sp041_20191205T1757.cdl

```diff
@@ -332,4 +332,46 @@ data:
  PLATFORM_MODEL = "Spray" ;
 
  WMO_IDENTIFIER = "4801948" ;
+
+group: CTD {
+
+  // group attributes:
+  		:long_name = "CTD sensor" ;
+  		:type = "CTD" ;
+  		:type_vocabulary = "http://vocab.nerc.ac.uk/collection/OGXXX/" ;
+  		:maker = "Seabird" ;
+  		:maker_vocabulary = "http://vocab.nerc.ac.uk/collection/OGXXX/" ;
+  		:model = "SBE 41CP" ;
+  		:model_vocabulary = "http://vocab.nerc.ac.uk/collection/OGXXX/" ;
```

Comment:
This is the NVS collection for sensor models - https://vocab.nerc.ac.uk/collection/L22/current/
To be adopted.

Reply:
Or maybe a subset like for Argo R27 (https://vocab.nerc.ac.uk/search_nvs/R27/)
This would guide user to a smaller list of sensors.

Reply:
We can create a subset of L22 (or more training on searching through L22). To be decided.

Reply:
Sounds interesting the idea of allowing more details. With both solutions, the change would be on the vocabularies, but the use here would be, in principle, the same, correct? So it doesn't affect this PR. @emmerbodc and @vturpin , please let me know if you agree to close this conversation which will allow one step forward in merging this PR. And let's open another issue later, suggesting which vocab to use.

Reply:
@emmerbodc and @vturpin , I think this was resolved with the following discussion. If you disagree, please re-open this discussion or just let me know.